source/sync/data-model/update-schema.txt

```diff
@@ -178,6 +178,13 @@ required property to both the client model and the Atlas schema.
 However, you should consider making the property optional to avoid the need to 
 modify existing Atlas documents. 
 
+.. note:: Required Properties Missing in a Schema Subset Will Default to Zero 
+
+   Clients can open the Realm with a subset of a schema that doesn't 
+   include a required property. In this case, the server populates the 
+   missing required value field with a zero or blank value
+   (like 0, "", or 0.0), depending on the property type.
+
```

Comment:
I can't comment directly, but the section below this - "Modify Existing Atlas Documents" - describes how you have to update existing documents with a new required property or they don't sync with the client. 

Is that still true with this new required property behavior? i.e. does the server populate missing required value fields only for _new_ documents, or does it also retroactively apply the missing default for documents that are already in a collection?

If it's true only for _new_ documents, we should probably add that in the note. Maybe something like "When you sync objects from the client that are missing a required property, the server populates the missing required value field with..."

And then maybe add clarification in the section below, too, that this behavior doesn't apply to existing documents in a collection, and they still have to populate those with default values or they don't sync.

OR if they no longer have to add default values to existing documents to get them to sync, we should probably delete the section below?

Reply:
It's only for new documents! I think specifying in the note will suffice. Wouldn't want to convolute the next section for such a small case. Let me know what you think of the update!

Reply:
Perfect, that makes sense. Thank you for clarifying!